pyproject.toml

```diff
@@ -32,3 +32,4 @@ build-backend = "poetry.core.masonry.api"
 sheets2linkml = "schemasheets.schemamaker:convert"
 linkml2sheets = "schemasheets.schema_exporter:export_schema"
 sheets2project = "schemasheets.sheets_to_project:multigen"
+generate-populate = 'schemasheets.generate_populate:cli'
```

Comment:
remember, these commands may be installed at a system level. `generate-populate` is a very generic script name.

Can you choose something that fits with the existing conventions and is less generic? E.g. linkml2sheets-template?